### ISSUE TITLE
[FEATURE] multisearch with multiple results per server

### ISSUE THREAD

**Issue by g-sloth** [OCSInventory-NG/OCSInventory-ocsreports#1253]
Hey guys!

First of all - very nice software that exactly fits my needs.

I just have one problem with the multisearch feature:

If I go to "Inventory > Search with various criteria" and search for 
_"All Software : Publisher"  different "Microsoft"_ I only get 1 result per Server.

What I'd like to see is every Software that meets this criteria together with the hostname. 
To illustrate what I'd like to achieve I build an SQL Query that does that (for simplicity i only selected 3 Columns):

```
SELECT sn.NAME AS Softwarename, h.NAME AS Hostname, sp.PUBLISHER AS Publisher

FROM software s

LEFT JOIN software_name sn

ON s.NAME_ID = sn.ID

LEFT JOIN software_publisher sp

ON s.PUBLISHER_ID = sp.ID

LEFT JOIN hardware h

ON s.HARDWARE_ID = h.ID

WHERE

sp.PUBLISHER NOT LIKE "Microsoft%";
```

Is this possible to get with the Webinterface?

Greetings

**Comment by Zythyx** [comment]
Agreed! I'm looking to find which monitors are connected to which computers and I can only see one monitor on each device through "Inventory > Search with various criteria", even though most of our machines have 2 or 3 monitors connected. This seems like a large oversight
